### DISCUSSION TITLE
Evaluate release-drafter

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-m2e/m2e-core#576] [Ideas]
Currently we write the release notes as part of the change on a rather manual process.

I recently came along a workflow that could automate this process based on the PRs from the repository:

https://github.com/release-drafter/release-drafter

**Comment by laeubi** [comment]
here is how it looks like:
https://github.com/codehaus-plexus/modello/releases/tag/modello-2.0.0

**Comment by HannesWell** [comment]
Github provides a similar feature (but it looks like it has fewer attributes):
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

The release notes generated by GitHub could then be used to generate a GH-release using:
https://github.com/marketplace/actions/gh-release

The release-drafter could be used to feed:
https://github.com/marketplace/actions/changelog-updater 

It depends if we want to keep the CHANGELOG.md file or if we provide only GH releases. It would also possible to do both and to refer to the correspending CHANGELOG.md entry from the GH release.
